packages/components/src/draggable/stories/index.tsx

```diff
@@ -38,7 +39,7 @@ const DefaultTemplate: ComponentStory< typeof Draggable > = ( args ) => {
 		<div>
 			<p>Is Dragging? { isDragging ? 'Yes' : 'No!' }</p>
 			<div
-				/* eslint-disable no-restricted-syntax, eslint-comments/disable-enable-pair */
+				/* eslint-disable-next-line no-restricted-syntax */
 				id="draggable-example-box"
```

Comment:
Duplicate ids are not going to work in Docs view (`?path=/docs/components-draggable--default`) where all stories are rendered in the same document.